app/assets/javascripts/components/student/student_comments.es6.jsx

```diff
@@ -46,6 +68,7 @@ class StudentComments extends Component {
       <div style={style}>
         <span style={this.styles.title}> Student Comments </span>
         {this.renderStudentComments()}
+        <Clickable func={this.showCreateOverlay.bind(this)}>Add Comment</Clickable>
```

Comment:
Change this to arrow syntax, hit an enter after the `>`, and add braces and quotes arround `Add comment`!